frontend/src/Settings/Profiles/Metadata/SecondaryTypeItems.js

```diff
@@ -0,0 +1,87 @@
+import PropTypes from 'prop-types';
```

Comment:
I think this was pulled from Readarr as it wasn't needed?

Reply:
Incoming commit